python/pfs/drp/stella/mergeArms.py

```diff
@@ -151,7 +155,21 @@ def runDataRef(self, expSpecRefList):
         spectra = [[dataRef.get("pfsArm") for dataRef in specRefList] for
                    specRefList in expSpecRefList]
         lsfList = [[dataRef.get("pfsArmLsf") for dataRef in specRefList] for specRefList in expSpecRefList]
-        pfsConfig = expSpecRefList[0][0].get("pfsConfig")
+        if self.config.pfsConfigFile:
+            try:
+                pfsDesignId, visit0 = re.split(r"[-.]", os.path.split(self.config.pfsConfigFile)[1])[1:-1]
+
+                pfsDesignId = int(pfsDesignId, 16)
+                visit0 = int(visit0)
+            except ValueError:
+                pfsDesignId = 666
+                visit0 = 0
+
```

Comment:
Is there a physical pfsDesign file available with that designId?

Reply:
In general not, but it doesn't actually matter.  The only reason for needing them is to make the pfsConfig I/O API happy